index.bzl

```diff
@@ -0,0 +1,13 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+""" Public API surface is re-exported here.
+
+This API is exported for users building angular material from source in
```

Comment:
Nit: for sake of consistency can you write `Angular Material` instead of `angular material`?

Reply:
👍